lib/osf-components/addon/components/node-metadata-form/styles.scss

```diff
@@ -11,7 +11,8 @@
         color: $color-link-black;
     }
 
-    div:not(.resource-edit) {
+    div:not(.resource-edit),
+    div:not(.metadata-description-field) {
```

Comment:
Would changing this margin into a padding keep you from having to complicate this CSS with this extra not and disabling the style lint rule below?

Reply:
Not in the options that I have tried.

Reply:
You mean you tried it and it didn't work, or you never tried it? Because changing margin to padding in the dev console _seems_ to fix the problem in a simpler way.

Reply:
<img width="946" alt="Screenshot 2023-02-03 at 9 47 43 AM" src="https://user-images.githubusercontent.com/6599111/216632861-2043b5d6-aacc-46cc-8d24-ab9990bdb5ea.png">

Reply:
Which line would you like changed and how?